server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const cors = require('cors');
+const mongoose = require('mongoose');
+
+const noteController = require('./notes/noteController');
+
+const server = express();
+
+server.use(cors({ origin: 'http://localhost:3000'}));
+server.use(express.json());
+
+server.get('/', (req, res) => {
+    res.status(200).json({ api: 'running...' });
+});
+
+server.use('/note', noteController);
+
+const port = process.env.PORT || 5000;
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/lambdanotesdb', {}, (error) => {
```

Comment:
Have you gotten a Mongo server set up on mLab yet? I like to set mine up on mLab from the start -- that way all my test data persists in the same place, and I don't have to worry about it later. It's super simple, let me know if you need help 👍